spring-integration-redis/src/main/java/org/springframework/integration/redis/inbound/RedisStoreMessageSource.java

```diff
@@ -143,16 +139,4 @@ public String getComponentType() {
 		return "redis:store-inbound-channel-adapter";
 	}
 
-	public RedisStore getResource() {
-		return this.resourceHolder.get();
-	}
-
-	public void afterCommit(Object object) {
-		this.resourceHolder.remove();
-	}
-
-	public void afterRollback(Object object) {
-		this.resourceHolder.remove();
-	}
-
```

Comment:
Yeah... Thought about that. But they are out of use anyway and does literally nothing because this `this.resourceHolder` is never set.
I can put a Migration Guide note about these methods removal since it is really doesn't break anything unless you use them for some "void" calls 🤷